packages/tocco-ui/src/Typography/Typography.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import {StyledSpan} from './StyledTypography'
+import {getTextOfChildren} from '../utilStyles'
+
+/**
+ * Utilize only React components (e.g. <Span>) instead of pure html tags (e.g. <span>)
+ * ensure coherent typography and prevent css leaking. Default Fonts are loaded
+ * automatically. When other fonts are defined it is required to load font files
+ * manually and to define corresponding @font-face.
+ */
+const Span = props => {
```

Comment:
Component name does no match file name

Reply:
This is a workaround. 

My first approach was to declare Span in Misc.js but I need a file which has the same name like the package to be able to load example file.

My second approach was to declare everything from Misc.js in Typography.js but it is only allowed to have one component in main file (if I remember right).

Both approaches should be related to tocco-ui-showcase implementation. 

I am open for other solutions. Maybe it is the way to code currently.